### DISCUSSION TITLE
Point-in-time / survivorship-bias-free data

### DISCUSSION THREAD

**Discussion by stevenwong** [ranaroussi/yfinance#1182] [General]
Hi all,

From what I understand, I can query YF for the latest information of a stock but this is only available for the current set of stocks. Any idea on how to do it for stocks that existed in the past? e.g., stocks that were taken over or died?

**Comment by BradKML** [comment]
Oh that might be tough but this could be hot-wired around (til 2019) https://github.com/teddykoker/survivorship-free-spy/tree/master/survivorship-free https://teddykoker.com/2019/05/creating-a-survivorship-bias-free-sp-500-dataset-with-python/
Extra blogs to note: https://archive.ph/NsoX5 https://archive.ph/hgnqd https://archive.ph/mUTlF
